keyloop/api/v1/identity.py

```diff
@@ -2,12 +2,13 @@
 
 import marshmallow
 from cornice.resource import resource
-from pyramid.httpexceptions import HTTPNoContent
-from pyramid.security import Everyone, Allow
+from pyramid.httpexceptions import HTTPNoContent, HTTPNotFound 
+from pyramid.security import forget, Everyone, Allow
```

Comment:
Esses imports não precisam
```suggestion
from pyramid.httpexceptions import HTTPNoContent
from pyramid.security import Everyone, Allow
```